packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 2.3.0
 
+* Adds the ability to request map renderer for Android.
```

Comment:
This change should be limited to the Android implementation package; nothing in the app-facing package should change (except for a README note, but that can be a follow-up PR).